include/oneapi/dpl/pstl/hetero/dpcpp/utils_ranges_sycl.h

```diff
@@ -21,7 +21,6 @@
 
 #include "../../utils_ranges.h"
 #include "../../iterator_impl.h"
-#include "../../glue_numeric_defs.h"
```

Comment:
I suggest including `execution_sycl_defs.h` if we remove `glue_numeric_defs.h`. The later transitively includes  `execution_sycl_defs.h`, which defines `__hetero_tag` and  `device_policy` used in the current file.

Reply:
Done.